src/content/configuration/configuration-languages.md

```diff
@@ -135,14 +135,14 @@ In the example below JSX (React JavaScript Markup) and Babel are used to create
 First install the necessary dependencies:
 
 ``` bash
-npm install --save-dev babel-register jsxobj babel-preset-es2015
+npm install --save-dev babel-register jsxobj @babel/preset-env
```

Comment:
`babel-register` is v6 and `@babel/preset-env` is v7 - you should not mix the two

Reply:
@antonk52 Please update, thanks @asapach !

Reply:
if I understood correctly, this should be as below, is that correct?

```
npm install --save-dev @babel/core @babel/register jsxobj @babel/preset-env
```

Reply:
If you're using v7, then you need to set `presets` below as `@babel/preset-env` as well: https://babeljs.io/docs/en/babel-preset-env